init.lua

```diff
@@ -14,6 +33,26 @@ function craftguide:get_recipe(item)
 	return item
 end
 
+function craftguide.extract_groupnames(groupname)
+	if string.sub(groupname, 1, 6) == "group:" then
```

Comment:
I'm not saying it's bad practice, just that you have the extra `self` that's passed implicitely compared to `foobar:find`. Stylistically, it's also shorter.